client/coral-admin/src/routes/Moderation/components/UserDetail.js

```diff
@@ -52,6 +50,16 @@ export default class UserDetail extends React.Component {
     this.props.changeStatus('rejected');
   }
 
+  showCopied() {
```

Comment:
I feel like the whole logic of managing the Copy button shouldn't be in this component, but rather outside of this component.